data.sql

```diff
@@ -1,9 +1,23 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, weight_kg)
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020-02-03', null, null, 10.23),
          ('Gabumon', '2018-11-15', 2, '1', 8),
          ('Pikachu', '2021-01-07', 1, '0', 15.04), 
          ('Devimon', '2017-05-12', 5, '1', 11);
 
 SELECT * FROM animals;  -- to verify inserted data
+
+-- Add a new column to the table
+ALTER TABLE animals
+ADD COLUMN species VARCHAR(80);
+
```

Comment:
- [ ] Please ensure that everything that creates or changes the table structure should be saved in the `schema.sql` file. :pray:

Reply:
This issue is fixed now